static/app/components/avatarChooser.tsx

```diff
@@ -105,10 +105,10 @@ class AvatarChooser extends Component<Props, State> {
   }
 
   handleSuccess(model: Model) {
+    addSuccessMessage(t('Successfully saved avatar preferences'));
     const {onSave} = this.props;
     this.setState({model});
     onSave(model);
-    addSuccessMessage(t('Successfully saved avatar preferences'));
```

Comment:
That's odd... any ideas why this doesn't get hit?

Reply:
Did some console logging, and it seems like nothing gets reached after `onSave(model);` -- possibly because it returns?

Reply:
👀 That shouldn't happen unless an exception is thrown in `onSave` -- let's pair on this a bit tomorrow?